### ISSUE TITLE
Abnormal circulating Complement Factor H Related Protein 1 concentration

### ISSUE THREAD

**Issue by JillMiller** [obophenotype/human-phenotype-ontology#5994]
**Preferred term label:** Abnormal circulating Complement Factor H Related Protein 1 concentration

**Synonyms** Abnormal circulating CFHR1

**Definition (free text, please give PubMed ID)** Concentration of Complement Factor H Related Protein 1 in the blood is outside of the normal range, PMID: 27814381

**Parent term (use hpo.jax.org/app)** Abnormal circulating protein level HP:0010876

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Decreased circulating Complement Factor H Related Protein 1 concentration

**Synonyms** Decreased circulating CFHR1

**Definition (free text, please give PubMed ID)** Concentration of Complement Factor H Related Protein 1 in the blood is below of the normal range

**Parent term (use hpo.jax.org/app)** Abnormal circulating Complement Factor H Related Protein 1 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by JillMiller** [comment]
**Preferred term label:** Increased circulating Complement Factor H Related Protein 1 concentration

**Synonyms** Increased circulating CFHR1

**Definition (free text, please give PubMed ID)** Concentration of Complement Factor H Related Protein 1 in the blood is above of the normal range

**Parent term (use hpo.jax.org/app)** Abnormal circulating Complement Factor H Related Protein 1 concentration

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
adding terms
